bcipy/signal_model/mach_learning/cross_valid.py

```diff
@@ -51,67 +52,23 @@ def cost_cross_validation_auc(model, opt_el, x, y, param, k_folds=10,
             x_valid = fold_x[list_valid]
             y_valid = fold_y[list_valid]
 
+            try:
```

Comment:
comment inline on this

Reply:
fixed

Reply:
I think you pushed to the wrong repo. You should not be pushing to BitBucket anymore